config/jobs-chromeos.yaml

```
@@ -393,6 +393,19 @@ jobs:
       <<: *kbuild-gcc-10-x86-chromeos-params
       flavour: amd-stoneyridge
 
+  kselftest-acpi:
+    template: kselftest.jinja2
+    kind: test
+    params:
+      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/amd64'
```

```suggestion
      nfsroot: 'http://storage.kernelci.org/images/rootfs/debian/bookworm-kselftest/20240313.0/{debarch}'
```

https://github.com/kernelci/kernelci-core/pull/2504 now allows this.